pkgs/applications/misc/qdirstat/default.nix

```diff
@@ -3,7 +3,7 @@
 , makeWrapper, perlPackages, mkDerivation }:
 
 let
-  version = "1.6.1";
+  version = "1.7";
 in mkDerivation rec {
```

Comment:
```suggestion
in mkDerivation {
```

Reply:
I would do this instead
```nix
{ stdenv, fetchFromGitHub, qmake
, coreutils, xdg_utils, bash
, makeWrapper, perlPackages, mkDerivation }:

mkDerivation rec {
  pname = "qdirstat";
  version = "1.7";
```

Reply:
Generally `let ... in` is a bit better way over `rec`. See https://nix.dev/anti-patterns/language.html#rec-expression. But `mkDerivation rec {` is more widely used throw nixpkgs.

Reply:
> https://github.com/NixOS/nixpkgs/pull/93296#issuecomment-663763561
>
> https://nix.dev/anti-patterns/index.html is an "opinionated guide" that lists several patterns which are still allowed in this repo and some are used in almost all derivations.
>
> See https://discourse.nixos.org/t/avoid-rec-expresions-in-nixpkgs/8293/2 for some discussion on this.